packages/agoric-cli/lib/chain-config.js

```diff
@@ -14,6 +14,24 @@ export const BLOCK_CADENCE_S = 5;
 export const ORIG_BLOCK_CADENCE_S = 5;
 export const ORIG_SIGNED_BLOCKS_WINDOW = 100;
 
+// Rewrite the app.toml.
+export function finishCosmosApp({ appToml, portNum = '26657' }) {
+  const rpcPort = Number(portNum);
+  const app = TOML.parse(appToml);
+
+  app.proxy_app = 'kvstore';
+
+  // Offset the GRPC listener from our rpc port.
+  app.grpc.address = `0.0.0.0:${rpcPort + 9090 - 26657}`;
```

Comment:
This seems fragile.

Reply:
Would it be better if I added manifest constants?